src/components/ha-icon-next.ts

```diff
@@ -0,0 +1,23 @@
+import "@polymer/iron-icon/iron-icon";
+// Not duplicate, this is for typing.
+// tslint:disable-next-line
+import { HaIcon } from "./ha-icon";
+
+export class HaIconNext extends HaIcon {
+  public connectedCallback() {
+    this.icon =
+      window.getComputedStyle(this).direction === "ltr"
+        ? "hass:chevron-right"
+        : "hass:chevron-left";
+
+    super.connectedCallback();
```

Comment:
Please add a comment to each component why super is last.

Reply:
Done